src/google/protobuf/compiler/csharp/BUILD.bazel

```diff
@@ -6,6 +6,15 @@ load("@rules_cc//cc:defs.bzl", "cc_library", "cc_test")
 load("@rules_pkg//:mappings.bzl", "pkg_files", "strip_prefix")
 load("//build_defs:cpp_opts.bzl", "COPTS")
 
+cc_library(
+    name = "csharp_names",
+    hdrs = ["csharp_names.h"],
```

Comment:
While we're in cleanup mode, can we name this just `names.h`? The stuttering names (`compiler/csharp/csharp_xyz.h`) are legacy I think, other languages have moved away from these names, eg. for C++ in https://github.com/protocolbuffers/protobuf/pull/9669)

Reply:
sure, good idea